openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/RuleForm/RuleForm.tsx

```diff
@@ -37,6 +43,14 @@ const RuleForm: FC<RuleFormProps> = ({ ruleData, setRuleData }) => {
     []
   );
 
+  const [policyFunctions, setPolicyFunctions] = useState<Function[]>([]);
+
+  const [conditionOptions, setConditionOptions] = useState<
+    { label: string; value: string }[]
```

Comment:
Don't we have generatedTypes?

Reply:
No, This is for antd Autocomplete option component.